sdk/lib/3rdparty/stlport/CMakeLists.txt

```diff
@@ -51,6 +51,11 @@ target_compile_definitions(stlport PRIVATE
     _STLP_USE_EXCEPTIONS _DLL __USE_CRTIMP
     _BUILD_STLPORT NATIVE_CPP_INCLUDE=${REACTOS_SOURCE_DIR}/sdk/include/c++)
 
+target_compile_definitions(stlport PUBLIC
+    _CRT_DECLARE_NONSTDC_NAMES
```

Comment:
Here however, the define shouldn't be set to 1 ? (in comparison to what's done in sdk/lib/crt/oldnames.cmake )

Reply:
fixed.

Reply:
Ah ok, if this is indeed what needs to be set, I'll recheck more carefully your PR for that, because iirc there are a bit more instances where it would need the =1.